File: Design_Pattern_and_Principles/MVCPatternExample/MVCTest.java
public class MVCTest {
    public static void main(String[] args) {
        Student model = new Student();
        model.setName("Utkarsh Raj");
        model.setId("STU-0179");
        model.setGrade("B");

        StudentView view = new StudentView();
        StudentController controller = new StudentController(model, view);
        controller.renderView();
        System.out.println("\nModifying student records through controller flow...");
        controller.updateStudentGrade("A+");
        controller.renderView();
    }
}

File: Week_1/Design_Pattern_and_Principles/MVCPatternExample/MVCTest.cs
using System;

public class MVCTest {
    public static void Main(string[] args) {
        Student model = new Student {
            Name = "Utkarsh Raj",
            Id = "STU-1024",
            Grade = "B"
        };

        StudentView view = new StudentView();
        StudentController controller = new StudentController(model, view);

        controller.RenderView();

        Console.WriteLine("\nModifying student records through controller flow...");
        controller.UpdateStudentGrade("A+");
        controller.RenderView();
    }
}